config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -50,3 +50,41 @@ gzip -1 "${DATA_DIR}/${BOARD}/chromiumos_test_image.bin"
 
 # Probably redundant, but better safe than sorry
 cleanup
+=======
```

Comment:
Seems like a leftover from a git conflict

Reply:
Yes, it's fixed now